packages/rn-tester/RNTester-macOS/AppDelegate.mm

```diff
@@ -152,6 +152,7 @@ - (Class)getModuleClassFromName:(const char *)name
 
 - (id<RCTTurboModule>)getModuleInstanceFromClass:(Class)moduleClass
 {
+  // Set up the default RCTImageLoader and RCTNetworking modules.
```

Comment:
For my own knowledge, did you see this like in `packages/rn-tester/RNTester/AppDelegate.mm` and manually copy it to the macOS RNTester here?

Reply:
Yes.